Assets/Scripts/ssjAug2018/DeilveryTarget/DeliveryTarget.cs

```diff
@@ -0,0 +1,54 @@
+﻿using pdxpartyparrot.Core.Actors;
+
+using UnityEngine;
+
+// TODO: Lots. Basic class implementation so I can reference this in the manager
+namespace pdxpartyparrot.ssjAug2018.DeliveryTargets
+{
+    [RequireComponent(typeof(Collider))]
+    [RequireComponent(typeof(AudioSource))]
+    public sealed class DeliveryTarget : NetworkActor {
+
+        /* Unsure if this is going to get animations.
+        #region Animations
+        [Header("Animations")]
+
+        [SerializeField]
+        private Animator _animator;
+        #endregion
+        */
+
+        public int DeliveriesRequired;
```

Comment:
public members are for sure bad. better to do something like this:

[SerializeField]
private int _deliveriesRequired = 1;

public int DeliveriesRequired => _deliveriesRequired;

That said, the number required might come from some other piece of data, or maybe randomized, depending on what design wants to do here.